### DISCUSSION TITLE
Syntax higlight: Differentiate comments (#) from multiline string (''') in .py files

### DISCUSSION THREAD

**Discussion by Laurencio** [tsujan/FeatherPad#579] [Ideas]
I think (I would prefer) multiline strings ('''  xxx ''') in .py files should be highlighted with the same colour as strings (Quotations), or with a different special colour, not with the same colour as comments (#).

**Comment by tsujan** [comment]
It's about different tastes. Kate highlights all ordinary quotes like comments, Qt Creator highlights comments and all kinds of quotes with the same color, while gEdit uses the same color for all kinds of quotes and highlights comments with another color.

FeatherPad makes the most distinctions without complicating its code too much (although its highlighting code *is* complex): it differentiates between ordinary quotes, single quotes and triple quotes and highlights the latter like comments.. But I'm not saying that's a rule to follow.
